torch/csrc/autograd/function.h

```diff
@@ -49,6 +51,47 @@ struct FunctionFlags {
   function_list next_functions;
 };
 
+namespace {
```

Comment:
I don't think we should have unnamed namespaces in header files. Use a `detail` namespace or something similar instead.

https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file

Reply:
Oops, Adam mentioned this but I forgot to fix it. Will do.